integration-cli/docker_cli_events_test.go

```diff
@@ -358,6 +359,10 @@ func (s *DockerCLIEventSuite) TestEventsFilterImageLabels(c *testing.T) {
 	// 2 events from the "docker tag" command, another one is from "docker build"
 	assert.Equal(c, len(events), 3, "Events == %s", events)
 	for _, e := range events {
+		// Ignore image create event emitted by "docker build"
+		if strings.Contains(e, "image create") {
+			continue
+		}
```

Comment:
This one may no longer be needed then as well I think?

Reply:
Good catch, removed